handler/population_types.go

```diff
@@ -54,6 +56,23 @@ func (h *PopulationTypes) Get(w http.ResponseWriter, req *http.Request) {
 	log.Info(ctx, "population types found", logData)
 	lpt := len(ptypes.Datasets)
 
+	var defaultDatasets []string
+
+	if r.DefaultDatasets {
```

Comment:
If the request is for `defaulfDatasets`, do we still need to go through all the logic from line 77 to the end of function? My understanding is that, we will just return the result of `GetDefaultDatasetPopulationTypes` encapsulated in `PopulationTypes`